src/main/java/net/dv8tion/jda/api/requests/restaction/pagination/EntitlementPaginationAction.java

```diff
@@ -85,21 +86,38 @@ public interface EntitlementPaginationAction extends PaginationAction<Entitlemen
      * @param guildId
      *        The guild id to filter by
      *
+     * @throws java.lang.IllegalArgumentException
+     *         If the provided {@code guildId} is {@code null}, empty or is not a valid snowflake
+     * @throws java.lang.NumberFormatException
+     *         If the provided {@code guildId} is negative or has a value greater than {@link Long#MAX_VALUE Long.MAX_VALUE}
+     *
      * @return The current EntitlementPaginationAction for chaining convenience
      */
     @Nonnull
-    EntitlementPaginationAction guild(@Nonnull String guildId);
+    default EntitlementPaginationAction guild(@Nonnull String guildId) {
```

Comment:
```suggestion
    default EntitlementPaginationAction guild(@Nonnull String guildId)
    {
```

Reply:
Weird how both of these showed for you, that was fixed hours before your review 🤔 . Resolved anyway